lib/aspects/govcloud-compatibility.test.ts

```diff
@@ -5,6 +5,7 @@ import * as lambda from "aws-cdk-lib/aws-lambda";
 import * as logs from "aws-cdk-lib/aws-logs";
 import { Annotations, Match, Template } from "aws-cdk-lib/assertions";
 import { GovCloudCompatibilityAspect } from "./govcloud-compatibility";
+import { test } from "@jest/globals";
```

Comment:
I understand this change is necessary to get things working at the moment. Can it be tracked as tech debt? The jest globals should just work and shouldn't need to be imported directly (notice that `describe` doesn't need this).

Reply:
@kylelaker-ccpo reverted change. I originally thought the test would break when pushed up, so I added the fix. Will capture in tech debt.